backend/infrahub/api/artifact.py

```diff
@@ -63,23 +64,23 @@ async def generate_artifact(
     branch_params: BranchParams = Depends(get_branch_params),
     account_session: AccountSession = Depends(get_current_user),
 ) -> None:
+    permission_manager = PermissionManager(account_session=account_session)
```

Comment:
Would be good to implement the permission manager as a Fastapi dependency directly
I did a quick search and fastapi support Sub-dependencies
https://fastapi.tiangolo.com/tutorial/dependencies/sub-dependencies/#first-dependency-dependable